app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,77 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.notice,
+.alert {
+  background-color: green;
+  padding: 10px;
+  text-align: center;
+}
+
+.alert {
+  background-color: red !important;
```

Comment:
- [ ] Let's make sure we don't use !important in your CSS. Using !important, however, is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets. When two conflicting declarations with the ! important rule are applied to the same element, the declaration with a greater specificity will be applied

Reply:
@arslanbisharat, I used it here to override the color of the alert message, that I used in the preview styling rules that way I can respect the DRY best practices. so in this situation is not a bad practice but enforcing the DRY rules
Thank you for your review